EduVPN-multiOS/Services/NotificationsService.swift

```diff
@@ -48,9 +48,15 @@ class NotificationsService: NSObject {
             identifier: CertificateExpiryNotificationAction.ignore.rawValue,
             title: NSString.localizedUserNotificationString(forKey: "Ignore", arguments: nil),
             options: [])
+        #if os(macOS)
+        // For some reason, macOS flips the order of buttons
+        let actions: [UNNotificationAction] = [ignoreAction, authorizeAction]
+        #else
+        let actions: [UNNotificationAction] = [authorizeAction, ignoreAction]
+        #endif
```

Comment:
It does so for good reason. Don't fight the system.

Reply:
I'm not strongly in favour of either order, so if you feel strongly that macOS is doing the right thing by flipping the order, I'm okay with that.